README.md

```diff
@@ -7,29 +7,34 @@ Node.js tools for Visual Studio 2017 is developed and managed here.
 
 
 Check out our documentation on [the NTVS wiki](https://github.com/Microsoft/nodejstools/wiki) 
-and our (old) [feature overview](https://channel9.msdn.com/events/Visual-Studio/Connect-event-2015/801) video on Channel 9.
+<!--and our (old) [feature overview](https://channel9.msdn.com/events/Visual-Studio/Connect-event-2015/801) video on Channel 9.
+-->
 
 Feel free to file issues or ask questions on our [issue tracker](http://github.com/Microsoft/nodejstools/issues), 
 and we welcome code contributions - see 
 [Contributing](https://github.com/Microsoft/nodejstools/blob/master/.github/CONTRIBUTING.md) for information. 
 
-## Visual Studio 2017 installation
+## Visual Studio 2019 installation
 
 The Node.js development workload is available as part of Visual Studio 2017 Community, Professional 
```

Comment:
2019?

Reply:
yes..  the link for normal VS installer went out to VS 2019 download and when you download 2019 you can install the node tools. And I made a saw change as I saw 2017 was still in that description

Reply:
No, I mean the paragraph contains the text `2017`. I think we should change that to `2019` as well.

Reply:
Oh yes I fixed that in the new push!